versions/latest/modules/en/pages/install/methods.adoc

```diff
@@ -34,11 +39,11 @@ chmod +x install.sh
 
 === Installation
 
-The installation process defaults to the latest RKE2 version and no other qualifiers are necessary. However, if you want to specify a version, you should set the `INSTALL_RKE2_CHANNEL` environment variable. An example below:
+Run the installer, where `INSTALL_RKE2_ARTIFACT_URL` is the Prime Artifacts URL and `INSTALL_RKE2_CHANNEL` is the most recent stable version of RKE2. If you want to specify a version, set the `INSTALL_RKE2_CHANNEL` environment variable.
 
 [,bash]
 ----
-INSTALL_RKE2_CHANNEL=latest ./install.sh
+sudo INSTALL_RKE2_ARTIFACT_URL=<PRIME-ARTIFACTS-URL>/rke2 INSTALL_RKE2_CHANNEL="latest" INSTALL_RKE2_CHANNEL=latest ./install.sh
```

Comment:
This is not 100% accurate. If you want to specify a version, you should use the envvar `INSTALL_RKE2_VERSION` instead. `INSTALL_RKE2_CHANNEL="latest"` will give you the latest version